### PR TITLE
Added optional attribute for embedded; when enabled, the advertising plugin is disabled

### PR DESCRIPTION
**Pull request by spra85**
@collin @daytonn @kand ... Queueing this up for a Monday review.

The purpose of the embedded attribute is to disable the JWPlayer advertising plugin for our own internal in-line article embedded players.

### CODE REVIEW

File: elements/bulbs-carousel/bulbs-carousel.test.js [hunk 1]
```diff
@@ -98,7 +98,7 @@ describe('<bulbs-carousel>', () => {
     });
 
     context('called twice', () => {
-      it.only('does not create a second track', () => {
+      it('does not create a second track', () => {
```
Comment (spra85) [inline elements/bulbs-carousel/bulbs-carousel.test.js]: This was causing the test suite to run only this test and skip all others.  So, yeah...

**Comment by collin** [comment]
Looks good to me.

**Comment by kand** [comment]
👍

File: elements/bulbs-video/components/revealed.js [hunk 2]
```diff
@@ -97,6 +97,10 @@ export default class Revealed extends React.Component {
       videoMeta.player_options.defaultCaptions = true;
     }
 
+    if (this.props.embedded) {
+      videoMeta.player_options.embedded = true;
+    }
+
```
Comment (daytonn) [inline elements/bulbs-video/components/revealed.js]: I think `this.props.embedded` is already boolean so you could just `videoMeta.player_options.embeded = this.props.embeded`, right? Is there something else I'm missing?

**Comment by daytonn** [comment]
other than that extra check, looks good 👍

File: elements/bulbs-video/components/revealed.js (hunk 2 above)
Comment (daytonn) [inline elements/bulbs-video/components/revealed.js]: you could do the coercion trick if you're paranoid `videoMeta.player_options.embedded = !!this.props.embedded`

Comment (spra85) [inline elements/bulbs-video/components/revealed.js]: You're right, @daytonn.  Tests pass, so do the examples.  Updated it

**Pull request merged**